src/server/object_services/md_store.js

```diff
@@ -1058,15 +1083,15 @@ class MDStore {
      * @param {nb.ObjectMD} obj
      * @returns {Promise<nb.ID[]>}
      */
-    async find_parts_chunk_ids(obj, none_deleted_only) {
+    async find_parts_chunk_ids(obj) {
         const find = {
             obj: obj._id,
+            deleted: null,
         };
-        if (none_deleted_only) find.deleted = null;
         return this._parts.col().find(find, {
```

Comment:
Can you add a hint to this query to make sure it uses the index we intended it to use?